contracts/RelayHub.sol

```diff
@@ -300,7 +307,7 @@ contract RelayHub is IRelayHub {
         uint256 totalInitialGas,
         uint256 maxPossibleGas
     )
-    external
+    external whenNotPaused
```

Comment:
Just noticing this. This function should not be external, right? This is probably too problematic